pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: sass
-version: 1.49.0
+version: 1.49.1
```

Comment:
Let's make this a -dev version for now, since we're still in the process of setting up the Embedded Sass infrastructure.

Reply:
Right, good call.